spec/controllers/staff/activity_historical_events_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Staff::ActivityHistoricalEventsController do
+  let(:user) { create(:delivery_partner_user, organisation: organisation) }
+  let(:organisation) { create(:delivery_partner_organisation) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    allow(controller).to receive(:logged_in_using_omniauth?).and_return(true)
+  end
+
+  describe "#show" do
+    let(:activity) { build_stubbed(:project_activity) }
+    let(:grouper) { instance_double(Activity::HistoricalEventsGrouper, call: double)}
```

Comment:
StandardRB doens't like this line

```suggestion
    let(:grouper) { instance_double(Activity::HistoricalEventsGrouper, call: double) }
```

Reply:
thks, should have noticed that one